pkg/arvo/sys/vane/clay.hoon

```diff
@@ -2114,7 +2114,8 @@
       ::
       =.  fiz  *melt
       ((slog [leaf+"clay: fuse failed, missing {<bec>}"]~) ..take-fuse)
-    ?>  (~(has by sto.fiz) bec)
+    ?.  (~(has by sto.fiz) bec)
+      ((slog [leaf+"clay: got strange fuse response {<bec>}"]~) ..take-fuse)
```

Comment:
Here and above on 2116 and 2099, should be `<(en-beam bec ~)>` so that it prints nicer.